graphdatascience/tests/integration/test_graph_object.py

```diff
@@ -154,3 +154,21 @@ def test_with_graph(gds: GraphDataScience) -> None:
         assert TempG.exists()
 
     assert not gds.graph.exists(temp_graph_name)["exists"]
```

Comment:
the test above still works so this is not breaking :)

Reply:
excellent. it's a good indicator that you mimicked the Tuple type. I'd like to make sure we thoroughly analyse that fact though, studying all aspects of the Tuple type. Chances are that someone is using it in a different way than we do in this repo.